### DISCUSSION TITLE
Replacing the default remote skin

### DISCUSSION THREAD

**Discussion by lead2crm** [GerryDazoo/Slinger#595] [Q&A]
I just now finally got my Slinger working and I see there are Tivo remote skins, which I'd like to have instead of the default one I have.
I have a Tivo T6 (not sure if the model truly matters as I'm truly just looking for the core features and better design)

How do I install one of the remotes supplied in the Custom Remoter folder? One would think I would just overwrite the remote.txt in the Slinger-main folder; however, the custom remotes are a mixture of .js, .txt and .html files, so I'm a bit lost; however, I have a lot of experience with these (since 1996), but I dont understand what action I have to take to tell Slinger to use the right files?

**Comment by f520k510** [comment]
Change the line "remote=remote.txt" to "tivoremote.html" in the config.ini and extract the tivoremote.zip to the folder where your config.ini is.
You may have to modify tivoremote.html with notepad to make the necessary changes to reflect the correct keycodes, but without modifying, the basic buttons should work.

**Reply by VernHausmann** [reply]
If you have an iPhone try "Peanut Remote for TiVo DVR"
You have to forward port 31339 from your TiVo then manually configure network settings on Peanut with your WAN IP and the port number( I have 4 TiVo's so the external ports on mine are 31336,31337,31338 and 31339 so I can control the one I want)
It works great and looks good, if you turn the phone sideways it gives you a keyboard for searches.
  I don't have an android so I don't know if it's available for them but it really works great on iPhone. 

 Thanks Gerry for all that you have done to save our slingboxes
